release_notes/44-release-notes.adoc

```diff
@@ -0,0 +1,572 @@
+
+In the integration form, select *Use container IAM role* to enable STS role assume that applies to Amazon EKS and {osp} clusters. 
+
+For Google Artifact Registry, Google Container Registry, Google Security Command Center and Google Cloud Storage integrations, support for authentication through workload identity federation has been added. 
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations*, and then select the integration type, for example, *Google Artifact Registry*. Select an existing integration. Alternatively, click *New integration* to create a new integration.
+
+In the integration form, select *Use workload identity* to enable token federation that applies to Google GKE and {osp} clusters. 
+
+[id="cluster-discovery-by-using-cloud-source-integrations_{context}"]
+=== Cluster discovery by using cloud source integrations
+
+{product-title-short} 4.4 introduces a new cloud source integration type, which includes integration types for Paladin Cloud and {cloud-redhat-com}. Integrated cloud sources enable {product-title-short} to discover cluster assets from the connected accounts. The discovered clusters are matched against clusters already secured by Central. 
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters -> Discovered clusters* to view the current status of the discovered clusters.
+
+[id="short-lived-api-tokens-for-central_{context}"]
+=== Short-lived API tokens for Central
+
+{product-title-short} 4.4 introduces the option of issuing short-lived API tokens for interaction with the Central API.
+
+You can exchange the short-lived API tokens for an OpenID Connect (OIDC) identity token. This is useful for the authentication and authorization of machines in, for example, CI environments.
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations -> Machine access configuration* to create a configuration to enable exchanging OIDC identity tokens for short-lived {product-title-short}-issued tokens.
+
+[id="migration-to-stock-red-hat-openshift-sccs-during-manual-upgrade-by-using-roxctl-cli_{context}"]
+=== Migration to stock {osp} SCCs during manual upgrade by using roxctl CLI
+
+With {product-title-short} 4.4, the platform moves away from using custom security context constraints (SCCs). Instead, {product-title-short} services use {osp} SCCs, which ensures future proof operation and a consistent security posture for {product-title-short}.
+
+For more information, see xref:../upgrading/upgrade-roxctl.adoc#migrating-sccs-during-the-manual-upgrade_upgrade-roxctl[Migrating SCCs during the manual upgrade].
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+* The default memory requirement for `scanner-db` has been increased from 200MiB to 512MiB to prevent out of memory (OOMs) errors during database initialization when the memory pressure reaches the node.
```

Comment:
```suggestion
* The default memory requirement for `scanner-db` has been increased from 200MiB to 512MiB to prevent out-of-memory (OOM) errors during database initialization when the memory pressure reaches the node.
```